empress/recon_vis/plot_tools.py

```diff
@@ -37,36 +59,72 @@ def __init__(self, title, axes: Union[plt.Axes, None] = None):
             self.axis = axes
         self.axis.autoscale()
         self.axis.margins(0.1)
-        self.axis.axis("off")
+        self.axis.axis(OFF)
```

Comment:
Let's switch back and use `self.axis.axis("off")`. When I read `self.axis.axis(OFF)`, my first response was what is `OFF`.

Reply:
Fixed